src/vde_backend/utils.py

```diff
@@ -129,6 +129,8 @@ def get_last_registered_config():
         allow_filtering=False,
         tz="UTC"
     )
+    if len(latest_configs) == 0:
+        return None
```

Comment:
I think you will introduce error in the rest of the code because you don't handle the None in the other files. For instance, in sync_flukso.py, you do config.method but with these lines, you do None.method ? => Error